src/general/WorldGenerationSettings.cs

```diff
@@ -91,6 +91,9 @@ public enum LifeOrigin
     [JsonIgnore]
     public bool FreeGlucoseCloud => Difficulty.FreeGlucoseCloud;
 
+    [JsonIgnore]
+    public PatchMap.FogOfWarMode PatchMapExplorationMode => Difficulty.PatchMapExplorationMode;
```

Comment:
I think the FogOfWarMode is general enough concept to put the enum at the global level (not inside the PatchMap class).